### PR TITLE
add research paper about Kiva images

### PR DESCRIPTION
**Pull request by go-bears**
add new research paper to Kiva's research page

### CODE REVIEW

**Comment by go-bears** [comment]
no rush for this review, btw! always appreciate the extra 👀

File: src/pages/Build/Research.vue [hunk 1]
```diff
@@ -34,8 +34,16 @@ export default {
 	},
 	data() {
 		return {
-			years: [2018, 2017, 2016, 2015, 2014, 2013, 2012, 2011, 2010],
+			years: [2019, 2018, 2017, 2016, 2015, 2014, 2013, 2012, 2011, 2010],
 			papers: [
+				{	year: 2019,
```
Comment (mcstover) [inline src/pages/Build/Research.vue]: @go-bears please drop this first key/value to it's own line, then your build should pass.

File: src/pages/Build/Research.vue [hunk 2]
```diff
@@ -34,8 +34,16 @@ export default {
...
+					author: 'Tsai, Jeanne et al',
+					title: 'Culturally Valued Facial Expressions Enhance Loan Request Success',
+					publication: 'American Psychological Association',
+					/* eslint-disable max-len */ 
+					url: 'https://culture-emotion-lab.sites.stanford.edu/sites/g/files/sbiybj9351/f/culturallyvaluedfacialloansuccess.pdf',
+					/* eslint-enable max-len */			
+					},
```
Comment (mcstover) [inline src/pages/Build/Research.vue]: ...and move this closing bracket 1 tab back...

File: src/pages/Build/Research.vue (hunk 1 above)
Comment (go-bears) [reply src/pages/Build/Research.vue]: thanks for the review!

Comment (go-bears) [reply src/pages/Build/Research.vue]: changed!

File: src/pages/Build/Research.vue (hunk 2 above)
Comment (go-bears) [reply src/pages/Build/Research.vue]: Appreciate your time, @mcstover!  fixed now!

**Comment by go-bears** [comment]
thanks for the 👍  @emuvente @mcstover !

**Pull request merged**